src/NuGetGallery/Services/IContentObjectService.cs

```diff
@@ -12,6 +14,8 @@ public interface IContentObjectService
         ICertificatesConfiguration CertificatesConfiguration { get; }
         ISymbolsConfiguration SymbolsConfiguration { get; }
         ITyposquattingConfiguration TyposquattingConfiguration { get; }
+        IGitHubUsageConfiguration GitHubUsageConfiguration { get; }
```

Comment:
Hey don't forget to trim the local `GitHubUsage.v1.json` file to have just test data.